src/Xamarin.Android.Build.Tasks/Tasks/ComputeHash.cs

```diff
@@ -14,18 +14,28 @@ public class ComputeHash : Task {
 		[Required]
 		public ITaskItem [] Source { get; set; }
 
+		public bool CopyMetaData { get; set; }
+
```

Comment:
Could just be:
```csharp
public bool CopyMetaData { get; set; } = true;
```

And we can delete the ctor?

Reply:
I keep forgetting about that :)